### PR TITLE
[NTRN-556] bump github.com/cosmos/interchain-security to v2.0.0

### PR DESCRIPTION
**Pull request by sotnikov-s**
**task:** https://p2pvalidator.atlassian.net/browse/NTRN-556

### CODE REVIEW

**Comment by sotnikov-s** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/5402783847

File: go.mod [hunk 1]
```diff
@@ -154,6 +155,7 @@ require (
 )
 
 replace (
+	github.com/ChainSafe/go-schnorrkel v1.0.0 => github.com/ChainSafe/go-schnorrkel v0.0.0-20200405005733-88cbf1b4c40d
```
Comment (mmikev) [inline go.mod:161]: Why?

Comment (sotnikov-s) [reply go.mod:161]: because the mev package uses the `v0.0.0-20200405005733-88cbf1b4c40d` version and refuses to compile with the `v1.0.0` one

**Review by pr0n00gler — DISMISSED** [review]

**Review by foxpy — COMMENTED** [review]
Looks good, but I suspect it might be consensus breaking?

**Pull request closed without merging**